strings/traumas.json

```diff
@@ -1,112 +1,118 @@
 {
 	"brain_damage": [
-		"@pick(semicolon)IM A PONY NEEEEEEIIIIIIIIIGH",
-		"without oxigen blob don't evoluate?",
-		"@pick(semicolon)CAPTAINS A COMDOM",
-		"@pick(semicolon)@pick(george) @pick(mellens) is grifing me HALP!!!",
-		"can u give me @pick(mutations)?",
-		"THe saiyans screwed",
-		"Bi is THE BEST OF BOTH WORLDS>",
-		"@pick(semicolon)I WANNA PET TEH monkeyS",
-		"stop grifing me!!!!",
-		"SOTP IT#",
-		"shiggey diggey!!",
-		"@pick(semicolon)A PIRATE APPEAR",
-		"FUS RO DAH",
-		"fucking 4rries!",
-		"stat me",
-		">my face",
-		"roll it easy!",
-		"waaaaaagh!!!",
-		"red wonz go fasta",
-		"FOR TEH EMPRAH",
-		"lol2cat",
-		"dem dwarfs man, dem dwarfs",
-		"SPESS MAHREENS",
-		"hwee did eet fhor khayosss",
-		"lifelike texture ;_;",
-		"luv can bloooom",
-		"PACKETS!!!",
-		"port ba@pick(y_replacements) med!!!!",
-		"REVIRT GON CHEM!!!!!!!!",
-		"youed call her a toeugh bithc",
-		"closd for merbegging",
-		"@pick(semicolon)pray can u @pick(create_verbs) @pick(create_nouns)???",
-		"GEY AWAY FROM ME U GREIFING PRICK!!!!",
-		"ur a fuckeing jackass!",
-		"@pick(semicolon)HELP SHITECIRTY MURDERIN  MEE!!!",
-		"hwat dose tha @pick(random_gibberish) mean?????",
-		"@pick(semicolon)CAL; TEH SHUTTLE!!!!!",
-		"wearnig siNGUARLTY is.... FINE haHAAA",
-		"@pick(semicolon)AI laW 22 Open door",
-		"@pick(semicolon)this SI mY stATIon......",
-		"who the HELL do u thenk u r?!!!!",
-		"geT THE FUCK OUTTTT",
-		"H U G B O X",
-		"@pick(semicolon)CRASHING THIS STTAYTION WITH NIO SURVIVROS",
-		"@pick(servers) is down!! @pick(bug)",
+		"@pick(semicolon)I'M A PONY, NEEEEEEIIIIIIIIIGH",
+		"I love you so much...",
+		"@pick(semicolon)MEDBAY TO ME!! I AM DEAD!!",
+		"@pick(semicolon)@pick(role) is KILLING me HELP!!!",
```

Comment:
@pick(role) won't work, as it's not defined like semicolon is, search for semicolon in that same file